packages/api-bindings/package.json

```diff
@@ -10,24 +10,24 @@
   "type": "module",
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
```

Comment:
how types are exported now?

Reply:
inside each `dist` dir, a `declarations` dir is created with `.d.ts` files for all of the types

there is no need to reference this from the `package.json`

Reply:
how do packages that depends on these find them? Is `declarations` dir a standard de-factor in this space?

Reply:
nope, works a little bit differently.

Typescript when looking for types is checking the `types` in `package.json` if that's not avaiable he tries to find the types by looking into where `main` points (in this case to `dist/lens-protocol-api-bindings.cjs.js`). He then goes to that folder and tries to see if there is a file called `dist/lens-protocol-api-bindings.cjs.d.ts`. 

Preconstruct just assumes the second behavior and relies on typescript to find the type definition by following where `main` points.

The generated `dist/lens-protocol-api-bindings.cjs.d.ts` points under the hood to `delcarations` folder 
```
exort * from "./declarations/src/index";

```
That's it, hope it clarifies your question @cesarenaldi.